data_steward/analytics/cdr_ops/retraction_in_place_validation_sandbox.py

```diff
@@ -0,0 +1,305 @@
+  {% if domain_id != 'ehr' and domain_id != 'person' and domain_id != 'death' and domain_id != '' and days != '0' %}
+    and {{domain_id}}_id > 2000000000000000
+  {% elif domain_id != 'ehr' and domain_id != 'person' and domain_id != 'death' and domain_id != '' and days == '0' %}
+    where {{domain_id}}_id > 2000000000000000
+  {% endif %}
+  ''')
+
+    old_row_counts_queries_list = []
+    new_row_counts_queries_list = []
+    is_deidentified = str('deid' in dataset).lower()
+
+    for table in pid_table_list:
+        domain_id = ''
+        if 'ehr' in table:
+            domain_id = 'ehr'
+        elif 'person' in table:
+            domain_id = 'person'
+        elif 'death' in table:
+            domain_id = 'death'
+        else:
+            for name in domain_tables:
+                if name in table:
+                    if 'observation' in name:
+                        if 'observation_period' in table:
+                            domain_id = 'observation_period'
+                        else:
+                            domain_id = 'observation'
+                        break
+                    else:
+                        domain_id = name
+                        break
+
+        old_row_counts_queries_list.append(
+            table_row_counts_query.render(project=project_id,
+                                          dataset=dataset,
+                                          table_name=table,
+                                          count='old_minus_aian_row_count',
+                                          days='1',
+                                          domain_id=domain_id))
+
+        new_row_counts_queries_list.append(
+            table_row_counts_query.render(project=project_id,
+                                          dataset=dataset,
+                                          table_name=table,
+                                          count='new_row_count',
+                                          days='0',
+                                          domain_id=domain_id))
+
+    old_row_counts_union_all_query = '\nUNION ALL\n'.join(
+        old_row_counts_queries_list)
+
+    new_row_counts_union_all_query = '\nUNION ALL\n'.join(
+        new_row_counts_queries_list)
+
+    old_retraction_table_count_query = (
+        f'{rids_query}\n{old_row_counts_union_all_query}'
+        if is_deidentified == 'true' else
+        f'{pids_query}\n{old_row_counts_union_all_query}')
+
+    new_retraction_table_count_query = (
+        f'{rids_query}\n{new_row_counts_union_all_query}'
+        if is_deidentified == 'true' else
+        f'{pids_query}\n{new_row_counts_union_all_query}')
+
+    old_count = execute(client, old_retraction_table_count_query)
+    new_count = execute(client, new_retraction_table_count_query)
+
+    results = pd.merge(old_count,
+                       new_count,
+                       on=['table_id', 'source', 'domain'],
+                       how='inner')
+
+    conditions = [
```

Comment:
Can we add this to the beginning of `conditions` so that UNKNOWN and person tables will unconditionally be `OK`?
`        (results['domain'] == 'UNKNOWN') | (results['domain'] == 'person') |`

Reply:
Yes! I forgot to update it.